rmgpy/rmg/modelTest.py

```diff
@@ -182,7 +190,64 @@ def testMakeNewReaction(self):
                     counter += len(rxnList)
 
         self.assertEquals(counter, 3)
+    
+    def testThermodynamicFiltering(self):
```

Comment:
This is a good functional test since it tests that the methods obtain the expected outcome. It is not necessarily a unittest, since it does not ensure the output of each method is correct. Would it be possible to write unittests for thermoFilterDown and possibly the section of processNewReactions?